domains/Contact/ManageGroups/Web/Controllers/GroupController.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Contact\ManageGroups\Web\Controllers;
+
+use App\Contact\ManageGroups\Web\ViewHelpers\GroupShowViewHelper;
+use App\Http\Controllers\Controller;
+use App\Models\Group;
+use App\Models\Vault;
+use App\Vault\ManageVault\Web\ViewHelpers\VaultIndexViewHelper;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Inertia\Inertia;
+
+class GroupController extends Controller
+{
+    public function index(Request $request, int $vaultId)
+    {
+    }
```

Comment:
Is there a reason to keep this empty function?

Reply:
Well, I forgot to remove it :-) But I'll fill this method in the subsequent PR as we'll have to do the index anyway.